package.json

```diff
@@ -33,7 +33,10 @@
     "react-router-bootstrap": "^0.26.0",
     "react-router-dom": "^6.2.1",
     "react-scripts": "^5.0.0",
-    "web-vitals": "^2.1.4"
+    "web-vitals": "^2.1.4",
+    "i18next": "^21.6.12",
+    "i18next-browser-languagedetector": "^6.1.3",
+    "react-i18next": "^11.15.5"
```

Comment:
`npm install` creates diffs with the checked in `package-lock.json` file at the moment.

Reply:
Oh yeah I forgot about that. Ran it now. ✅